fetch-demo.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+output_tar=$1
+
+git clone kde:scratch/apol/fetch-demo-data || true
```

Comment:
Users might not have kde: defined in the gitconfig, use https://anongit.kde.org/